cmd/deploy/remote.go

```diff
@@ -187,7 +187,7 @@ func (rd *remoteDeployCommand) createDockerignoreIfNeeded(cwd, tmpDir string) er
 			return err
 		}
 
-		err = afero.WriteFile(rd.fs, fmt.Sprintf("%s/%s", tmpDir, dockerignoreName), dockerignoreContent, 0600)
+		err = afero.WriteFile(rd.fs, fmt.Sprintf("%s/%s", tmpDir, ".dockerignore"), dockerignoreContent, 0600)
```

Comment:
I think this was the only place where we being using `dockerignoreName` constant. Should we delete that cons?

Reply:
It is used just before this line but I think it is not necessary for now to make it constant if it is only used once